app/scenes/select_team/select_team.js

```diff
@@ -17,16 +17,18 @@ import {GlobalStyles} from 'app/styles';
 
 import logo from 'assets/images/logo.png';
 
-export default class SelectTeam extends React.Component {
+export default class SelectTeam extends PureComponent {
```

Comment:
I'd be careful about having a top level component as a `PureComponent` since I think that might prevent child components from re-rendering when they change